src/snmalloc/pal/pal_posix.h

```diff
@@ -130,7 +131,11 @@ namespace snmalloc
 #endif
       ;
 
+#ifdef PAGESIZE
+    static constexpr size_t page_size = PAGESIZE;
```

Comment:
This works on my PowerPC machine and passes CI, but I wonder if we want to do something like...

```suggestion
    static constexpr size_t page_size = max(Aal::smallest_page_size, PAGESIZE);
```

Reply:
Sorry, I messed up the force push.  Put it back in now